.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -204,12 +211,29 @@ jobs:
           echo "REMOTE_SERVICE_DEPLOYMENT_NAME=$(kubectl get deployments -n ${{ env.SAMPLE_APP_NAMESPACE }} --selector=app=remote-app -o jsonpath='{.items[0].metadata.name}')" >> $GITHUB_ENV
           echo "REMOTE_SERVICE_POD_IP=$(kubectl get pods -n ${{ env.SAMPLE_APP_NAMESPACE }} --selector=app=remote-app -o jsonpath='{.items[0].status.podIP}')" >> $GITHUB_ENV
 
-      - name: Verify pod Adot image
+      - name: Log pod Fluent Bit image ID
+        run: |
+          kubectl get pods -n amazon-cloudwatch -l k8s-app=fluent-bit -o json | \
+          jq '.items[0].status.containerStatuses[0].imageID'
+
+      - name: Log pod Adot image and save image to the environment
         run: |
           kubectl get pods -n ${{ env.SAMPLE_APP_NAMESPACE }} --output json | \
           jq '.items[0].status.initContainerStatuses[0].imageID'
 
-      - name: Verify pod CWAgent image
+          echo "NEW_ADOT_IMAGE"=$(kubectl get pods -n ${{ env.SAMPLE_APP_NAMESPACE }} -o json | \
+          jq '.items[0].status.initContainerStatuses[0].image') >> $GITHUB_ENV
+
+      - name: Verify ADOT Image changed if ADOT Patched
+        run: |
+          if [ ${{ inputs.appsignals-adot-image-name }} != "" ]; then
```

Comment:
I think it will be nicer if we add the if logic as part of the github action workflow:
Ex: 
```
- name: Verify ADOT Image changed if ADOT Patched
  if: ${{ inputs.appsignals-adot-image-name }} != ""
  run: ...
```

or we could combine it into one if statement: 
`if [ (first expression) && (second expression) ].`

Reply:
Yeap that's my fault, I am not familar with github action, will make upgrade soon